R/serialized_tests.R

```diff
@@ -158,27 +164,41 @@ style_empty <- function(text) {
 #' @describeIn test_transformer Transformations for indention based on curly
 #'   brackets only.
 style_indent_curly <- function(text) {
+
+  transformers <- list(
+    filler     = create_filler,
+    line_break = NULL,
+    space      =  partial(indent_curly, indent_by = 2),
+    token      = NULL,
+    eol        = strip_eol_spaces,
+    NULL
+  )
+
   text %>%
     compute_parse_data_nested() %>%
-    pre_visit(funs = c(create_filler,
-                   partial(indent_curly, indent_by = 2),
-                   strip_eol_spaces)) %>%
-
+    apply_transformers(transformers) %>%
     serialize_parse_data_nested()
 }
 
 
 #' @describeIn test_transformer Transformations for indention based on curly
 #'   brackets and round brackets.
 style_indent_curly_round <- function(text) {
+  transformers <- list(
+    filler     = create_filler,
+    line_break = NULL,
+    space      =  c(partial(indent_curly, indent_by = 2),
+                    partial(indent_round, indent_by = 2)),
+    token      = NULL,
+    eol        = strip_eol_spaces,
+    NULL
+  )
+
   text %>%
     compute_parse_data_nested() %>%
-    pre_visit(funs = c(create_filler,
-                   partial(indent_curly, indent_by = 2),
-                   partial(indent_round, indent_by = 2),
-                   strip_eol_spaces)) %>%
-
+    apply_transformers(transformers) %>%
     serialize_parse_data_nested()
+
 }
 
 #' @describeIn test_transformer Transformations for indention based on operators
```

Comment:
Regarding `style_op()`: Ok, I will do that in a separate PR then. Regarding EOL space stripping, I also thought about that. I think in terms of performance, it does not matter. But it might be more coherent to strip these in the serialisation step. Ok?

`transformers$space` is a unnamed list of space transformers, so are `transformers$line` and `transformers$eol` (which has only one function though). The distinction is made since between line manipulation and space manipulation, we need to set the multi-line attribute.